src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageItemViewModel.cs

```diff
@@ -284,6 +285,14 @@ public bool IsNotInstalled
             }
         }
 
+        public bool IsInstalledAndTransitive
+        {
+            get
+            {
+                return (Status == PackageStatus.NotInstalled && LatestVersion != null) || PackageLevel == PackageLevel.Transitive;
```

Comment:
I am trying to understand why would a package in `NotInstalled` state be considered as a transitive package.

Reply:
I use this property to show the install icon for a package, not to define if a package is transitive. 

![image](https://user-images.githubusercontent.com/43253759/161906954-0bcc2c83-8d0f-42c8-8b04-05f1eb562902.png)